src/library/dao/playlistdao.cpp

```diff
@@ -244,6 +244,26 @@ bool PlaylistDAO::isPlaylistLocked(const int playlistId) const {
     return false;
 }
 
+// TODO: Deduplicate with deletePlaylist?
```

Comment:
Yes, this would be nice. This will also allow to clan up the double transaction and emit(changed(playlistId)) in the new replaceTracksInPlaylist.